modules/docs/lib/widgetUtils.tsx

```diff
@@ -102,31 +157,69 @@ export const SymbolDialog = ({value}: SymbolDialogProps) => {
         >
           {value.displayName || value.name}
         </Dialog.Target>
+
         {renderTypeParameters(value.typeParameters)}
-      </>
-      <Dialog.Popper>
-        <Dialog.Card maxHeight="50vh" maxWidth="90vh">
-          <Dialog.CloseIcon />
-          <Dialog.Heading>{value.name}</Dialog.Heading>
-          <Dialog.Body>
-            <RenderContext.Provider value="table">
-              <IndentLevelContext.Provider value={0}>
-                {symbol ? (
-                  <SymbolDoc name={value.name} headingStart={3} hideHeading />
-                ) : (
-                  <>
-                    <p>Basic type information:</p>
-                    <pre>
-                      <code>{value.value}</code>
-                    </pre>
-                  </>
-                )}
-              </IndentLevelContext.Provider>
-            </RenderContext.Provider>
-          </Dialog.Body>
-        </Dialog.Card>
-      </Dialog.Popper>
-    </Dialog>
+
+        <Dialog.Popper>
+          <Dialog.Card maxHeight="50vh" maxWidth="90vh">
```

Comment:
Love the breadcrumbs! It does feel a little odd when the dialog changes size as you drill down, though:

https://user-images.githubusercontent.com/146020/223900595-4ab3ee09-08a7-4e3a-b8c2-f9af7ed70eab.mov

I wonder if setting fixed dimensions for the `Dialog.Card` would be an improvement 🤔

Reply:
oh interesting... my only worry is when the site is shrunk. let me play with it!

Reply:
Tabling this potential change for a future PR.